sdk/servicebus/azure-servicebus/README.md

```diff
@@ -11,10 +11,15 @@ Use the Service Bus client library for Python to communicate between application
 * Send and receive messages within your Service Bus channels.
 * Utilize message locks, sessions, and dead letter functionality to implement complex messaging patterns.
 
-Note: This is a preview release of the Python Service Bus SDK, and is not yet at feature parity with version 0.50.  Some functionality, such as topic and subscription utilization, will be arriving in upcoming Preview releases; Thank you for your patience and interest as we roll out these changes.
-
 [Source code](./) | [Package (PyPi)][pypi] | [API reference documentation][api_docs] | [Product documentation][product_docs] | [Samples](./samples) | [Changelog](./CHANGELOG.md)
 
+> **NOTE**: This document has instructions, links and code snippets for the **preview** of the next version of the `azure-servicebus` package
+> which has different APIs than the stable version (0.50). Please view the resources below for references on the existing library.
```

Comment:
When I see a version number < 1.0, I wouldn't think it's "stable". But if this is customary in Python I have no problem with it.

Reply:
This verbiage was stolen from JS, who I believe did have a stable release, so your point is apt.  Let's change it.
```suggestion
> which has different APIs than the current version (0.50). Please view the resources below for references on the existing library.
```